common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -132,6 +133,7 @@ void extractState(@NonNull final Bundle state) {
     public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
         final String methodName = "#onCreateView";
         final View view = inflater.inflate(R.layout.common_activity_authentication, container, false);
+        final String[] javascriptToExecute = new String[1];
```

Comment:
This is minor, but I'm unclear on the need for this to be an array reference.  Since it's never in use outside of your anonymous OnPageLoadedCallback, why can it not be local?

Reply:
Moved the array declaration inside onPageLoaded() callback. Thanks.